### DISCUSSION TITLE
Nested List

### DISCUSSION THREAD

**Discussion by yoursamlan** [mgmeyers/obsidian-kanban#99] [Ideas & Feature Requests]
Nested list for more compact view. 

eg. 
## ToDo for Today
- This list may contain sub-lists like:
### Session-1
    - [ ] Some Task
    - [ ] Some other Task
#### Session-2.1
        - [ ] Some Task
        - [ ] Some Other Task

and so on...
Also, Drag a list inside another list to create a nested list will be awesome.

**Comment by Friedolito** [comment]
When I understand this correctly, I realy favour this feature too. 
The Kanband layout is super useful to manage tasks. 
But when I want to add aditional information to a task (e.g. website, subtasks, idea) things get messy. 
A way to add information that can be hidden, would make it a lot more useful (at least in my case).

**Reply by yoursamlan** [reply]
Hi, sorry for replying late and also for the fuzzy description beforehand. I have drawn a draft outline of my "nested list" idea [Attached below]. Hope it helps.
![Nested List](https://user-images.githubusercontent.com/33586885/119459103-05170380-bd5b-11eb-9f87-a8a31bbc80a4.png)


And yeah, I do agree with you that minimalism can improve productivity. But, I think, adding a nested list feature may help to organize tasks even better. :)

Anyway, thanks a lot for your revert and I'm really grateful to you for this awesome plugin. 
:)

**Comment by firinael** [comment]
I added this as an issue, seeing as it was just a discussion topic.

**Reply by huyz** [reply]
where?

**Comment by instancer-kirik** [comment]
One could also implement this by having a checkable list item like to a separate list, but then the issue of link resolution and focus onClick of link

**Comment by Shiraz-V-Kamar** [comment]
I was creating new ideas and I made draft before finding this idea, 

if you would like, you could use this for ur description, other info...
---
Description:

**Overview:**
Currently, the Kanban plugin for Obsidian application offers an excellent visualization of tasks, aiding users in managing their projects effectively. However, to enhance its utility further, I propose integrating subtask functionality within the plugin.

**Feature Proposal:**
The suggested feature aims to allow users to create and manage subtasks directly within the Kanban plugin interface. Specifically, I suggest displaying these subtasks as nested elements under the title of the parent task card.

**Functionality:**

_Creating Subtasks_: Users should be able to create subtasks directly within the Kanban board interface. This could be achieved through a context menu option or a designated button within the card.
_Visibility:_ Subtasks should be prominently displayed under the title of the parent task card. This ensures that users can easily identify and manage related tasks within the same view.
_Editing and Tracking_: Users should have the ability to edit subtasks, mark them as complete, or update their status without leaving the Kanban board view. This streamlines task management and improves workflow efficiency.
_Hierarchy Support_: The proposed feature should support multiple levels of subtasks, allowing for complex project structures while maintaining clarity and organization within the Kanban board.

**Benefits:**

_Improved Task Management_: By incorporating subtask functionality directly into the Kanban plugin, users can streamline their project management process without relying on external tools or manual workarounds.
_Enhanced Visualization_: The addition of subtasks under parent task cards provides users with a comprehensive overview of project progress and dependencies within a single interface.
_Increased Productivity_: Seamless task management within the Kanban board reduces context switching and improves productivity by keeping users focused on their current workflow.

**Conclusion:**
Incorporating subtask functionality into the Kanban plugin for Obsidian application would significantly enhance its utility for users managing complex projects. This feature aligns with the plugin's goal of providing a streamlined and intuitive project management experience within the Obsidian environment.

Screenshot of proposed idea ⬇️ 
![Screenshot 2024-04-01 045438](https://github.com/mgmeyers/obsidian-kanban/assets/42828670/99d25564-32e8-44fd-8813-e1859e0dbb7b)
